### PR TITLE
Video explaining Common Directory

### PR DESCRIPTION
**Pull request by vannguyen141290**
I paired with @cqian12 (Charles) for this task on Trello:
- Loom video explaining all 9 components in _src>>components>>common_ and inserted the video into main README file. 

Loom[https://www.loom.com/share/e304d66df57a42b4871ed82a2aa98dd1](url)

### CODE REVIEW

**Review by stevenjhomem — APPROVED** [review]

**Comment by zach-morris-txt** [comment]
That was an excellent overview. I've been watching these videos to better understand those components myself.
I do have a question for you @cqian12 or @vannguyen141290:

**Comment by vannguyen141290** [comment]
> That was an excellent overview. I've been watching these videos to better understand those components myself. I do have a question for you @cqian12 or @vannguyen141290: At 3:20 in the video, you mention the "defaultSelectedKeys". Does it error out without it? Is it necessary or is it redundant code? And considering the redundancy elsewhere, do you guys have a list somewhere of things to come back to?

yep. it does not matter. good point. I'll modify it. thanks for the insight :D

File: README.md [hunk 1]
```diff
@@ -120,4 +120,7 @@ Remember that this project is licensed under the MIT license, and by submitting
 
 ## Documentation
 
-See [Backend Documentation](https://github.com/BloomTech-Labs/coder-heroes-be#readme) for details on the backend of our project.
\ No newline at end of file
+See [Backend Documentation](https://github.com/BloomTech-Labs/coder-heroes-be#readme) for details on the backend of our project.
+
+## Common Directory Explanation
+Loom[https://www.loom.com/share/e304d66df57a42b4871ed82a2aa98dd1]
```
Comment (zach-morris-txt) [inline README.md:126]: Very organized

**Review by zach-morris-txt — APPROVED** [review]
Your video was excellent. It was hard to decipher what those common components were intended for and your explanation greatly simplified it.

**Pull request merged**